default.nix

```diff
@@ -109,10 +112,11 @@ let
         pkg-def = plan-pkgs.pkgs;
         # The compiler referenced in the stack config
         compiler = (plan-pkgs.extras hackage).compiler or (pkg-def hackage).compiler;
+        patchesModule = ghcHackagePatches.${compiler.nix-name} or {};
```

Comment:
I think this is a reasonable default value.

Reply:
Well, I just made it the same as the one for the (mostly identical) stack function.